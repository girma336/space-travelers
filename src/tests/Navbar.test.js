import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import store from '../Redux/configureStore';
import NavBar from '../components/Navbar';

describe('Test snapshot componet of rocket', () => {
  it('Renders correctly of nav bar', () => {
    const tree = render(
      <BrowserRouter>
        <Provider store={store}>
          <NavBar />
        </Provider>
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
