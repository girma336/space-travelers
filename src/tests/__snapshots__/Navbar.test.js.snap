// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot componet of rocket Renders correctly of nav bar 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navbar"
      >
        <div
          class="logo"
          style="text-decoration: none;"
          to="/"
        >
          <img
            alt="Logo"
            class="logo-image"
            src="planet.png"
          />
          <h1
            class="nav-heading"
          >
            Space Travelers\` Hub
          </h1>
        </div>
        <div
          class="nav-links"
        >
          <ul
            class="nav-ul"
          >
            <li
              class="nav-li"
            >
              <a
                aria-current="page"
                class="nav-link active"
                href="/"
              >
                Rockets
              </a>
            </li>
            <li
              class="nav-li"
            >
              <a
                class="nav-link"
                href="/missions"
              >
                Missions
              </a>
            </li>
            <li
              class="nav-li"
            >
              <a
                class="nav-link"
                href="/myprofile"
              >
                My profile
              </a>
            </li>
          </ul>
        </div>
        <hr />
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="navbar"
    >
      <div
        class="logo"
        style="text-decoration: none;"
        to="/"
      >
        <img
          alt="Logo"
          class="logo-image"
          src="planet.png"
        />
        <h1
          class="nav-heading"
        >
          Space Travelers\` Hub
        </h1>
      </div>
      <div
        class="nav-links"
      >
        <ul
          class="nav-ul"
        >
          <li
            class="nav-li"
          >
            <a
              aria-current="page"
              class="nav-link active"
              href="/"
            >
              Rockets
            </a>
          </li>
          <li
            class="nav-li"
          >
            <a
              class="nav-link"
              href="/missions"
            >
              Missions
            </a>
          </li>
          <li
            class="nav-li"
          >
            <a
              class="nav-link"
              href="/myprofile"
            >
              My profile
            </a>
          </li>
        </ul>
      </div>
      <hr />
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
